koku/masu/util/gcp/gcp_post_processor.py

```diff
@@ -124,9 +135,11 @@ def _generate_daily_data(self, data_frame):
 
         # this parses the credits column into just the dollar amount so we can sum it up for daily rollups
         rollup_frame = data_frame.copy()
+        # FIXME: Change the credits data to be a dictionary to avoid deserializing
         rollup_frame["credits"] = rollup_frame["credits"].apply(json.loads)
         rollup_frame["daily_credits"] = 0.0
         for i, credit_dict in enumerate(rollup_frame["credits"]):
+            # FIXME: Remove chained assignment
             rollup_frame["daily_credits"][i] = credit_dict.get("amount", 0.0)
```

Comment:
Instead of adding a FIXME, we could just fix this:
```suggestion
        rollup_frame["daily_credits"] = rollup_frame["credits"].apply(lambda x: x.get("amount", 0.0))
```
```